WordPress/Resources/ar.lproj/Localizable.strings

```diff
@@ -8345,7 +8345,7 @@ translators: %s: Select control option value e.g: \"Auto, 25%\". */
 "Use icon button" = "استخدام زر الأيقونة";
 
 /* Footer text for Invite Links section of the Invite People screen. */
-"Use this link to onboard your team members without having to invite them one by one. Anybody visiting this URL will be able to sign up to your organization, even if they received the link from somebody else, so make sure that you share it with trusted peo" = "استخدم هذا الرابط لضمّ أعضاء فريقك دون الحاجة إلى دعوتهم واحدًا تلو الآخر. سيتمكن أي شخص يزور عنوان الموقع هذا من الاشتراك في مؤسستك، حتى إذا تلقى الرابط من شخص آخر، لذا تأكد من مشاركته مع أشخاص موثوق بهم.";
+"invite_people_invite_link_footer" = "استخدم هذا الرابط لضمّ أعضاء فريقك دون الحاجة إلى دعوتهم واحدًا تلو الآخر. سيتمكن أي شخص يزور عنوان الموقع هذا من الاشتراك في مؤسستك، حتى إذا تلقى الرابط من شخص آخر، لذا تأكد من مشاركته مع أشخاص موثوق بهم.";
```

Comment:
Fixed all of those manually (search-and-replace) even if those files will be overwritten during next code freeze then after next beta downloading the latest `Localizable.strings` for all locales from GllotPress.

The rationale being that even though those will be overwritten, manually fixing them here in the meantime will allow for a smooth transition between re-enabling the linter and having that key do the round trip to get fixed in GlotPress. We wouldn't want the linter being re-enabled creating blockers during the transition period between this PR being merged and them going to GlotPress and back.

See f3e2a952e26010d17efa2d11d85db3c16c34f276 for full rationale.

Reply:
>  even though those will be overwritten, manually fixing them here in the meantime will allow for a smooth transition

Always good approach to take 👍